controller/src/test/java/org/jboss/as/controller/OperationCancellationUnitTestCase.java

```diff
@@ -66,14 +66,15 @@
 import org.jboss.msc.service.StopContext;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Unit tests of {@link ModelControllerImpl}.
  *
  * @author Brian Stansberry (c) 2011 Red Hat Inc.
  */
-@org.junit.Ignore
+@Ignore("WFCORE-6158")
```

Comment:
Hi @ropalka , would you mind adding a short description to WFCORE-6158? It is unclear your intention with this Jira, I'm going to merge this PR since I understand WFCORE-6158 is in place to just investigate whether this test case must be removed, but I'll leave the Jira unresolved until it is clear what you want to do with this case.